ajax/libs/awesome-bootstrap-checkbox/package.json

```diff
@@ -2,8 +2,8 @@
   "name": "awesome-bootstrap-checkbox",
   "filename": "awesome-bootstrap-checkbox.min.css",
   "version": "1.0.0",
-  "description": "Font Awesome Bootstrap Checkboxes & Radios. Pure css way to make inputs look prettier.",
-  "homepage": "http://awesome-bootstrap-checkbox.okendoken.com/demo/index.html",
+  "description": "Font Awesome Bootstrap Checkboxes & Radios. Pure CSS way to make inputs look prettier.",
+  "homepage": "http://flatlogic.github.io/awesome-bootstrap-checkbox",
```

Comment:
@preeteshjain Thank you for your contribution.
Let's use `https` because there is more safety for browsing.
Thank you.

Reply:
Okay I will make the changes.